docs/sources/developers/http_api/authentication.md

```diff
@@ -0,0 +1,23 @@
+---
+canonical: /docs/grafana/latest/developers/http_api/authentication/
```

Comment:
I've not actually tested if we can omit the scheme and hostname from canonicals. I would guess it would be fine but perhaps it's safer to include it?

```suggestion
canonical: https://grafana.com/docs/grafana/latest/developers/http_api/authentication/
```